code/modules/surgery/bodyparts/robot_bodyparts.dm

```diff
@@ -115,18 +115,19 @@
 	damage_examines = list(BRUTE = ROBOTIC_BRUTE_EXAMINE_TEXT, BURN = ROBOTIC_BURN_EXAMINE_TEXT)
 	bodypart_flags = BODYPART_UNHUSKABLE
 
-/obj/item/bodypart/leg/left/robot/emp_act(severity)
+/obj/item/bodypart/leg/left/robot/emp_effect(severity, protection)
 	. = ..()
-	if(!. || isnull(owner))
-		return
+	if(isnull(owner))
```

Comment:
Why get rid of `!.`? Same goes for all of these.

Reply:
The robot limb will always be robotic, we really don't need to spend time checking the return value of the parent

Reply:
You should still keep it, there's no reason not to and it saves future time if we add some other `return FALSE` condition to `emp_effect`